vars/getSHHKeysPMM.groovy

```diff
@@ -0,0 +1,10 @@
+String call() {
+    // You need to add your keys to https://github.com/settings/keys
+    String sshKeys = ''
+    final List additional_keys = ['nikita-b', 'nailya', 'puneet0191'] // github users
```

Comment:
Why only these people and not other? It doesn't make sense to me.

Reply:
These are the people who asked. If anyone else wants, we will add :) 

I just don't want to add all people because it will slow down the build.

Reply:
Let's see how to overcome the issue.